.github/workflows/v_apps_and_modules_compile_ci.yml

```diff
@@ -119,6 +119,17 @@ jobs:
           echo "Execute Tests"
           cd /tmp/adventofcode && v run verify.v
 
+      - name: Build vlang/msgpack
+        run: |
+          echo "Install msgpack"
+          v install msgpack
+          echo "Build msgpack"
+          v ~/.vmodules/msgpack/
```

Comment:
No, that is a module, not a project, there is no need to build it.
You could potentially do: `v -shared ~/.vmodules/msgpack/` instead .